Bottom: c6266842fb1b99a6a6178d614405f2f2d10f18b1
Top:    f21db72b17960bc5c38145ae7d83634c97ab1b2c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-11-14 23:58:41 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
